### ISSUE TITLE
Proposal: Allow copying multiple files to the same destination via `::copy([...], dest)`

### ISSUE THREAD

**Issue by maowtm** [modus-continens/modus#188]
What do you think... \
This does make `::copy` polymorphic though

**Comment by mechtaev** [comment]
Yes, it makes sense to do after we add polymorphism, because Dockefiles also support copying multiple files.

**Comment by maowtm** [comment]
We can already do it since we don't have type checking yet, and basically when we add type checking we will need to be able to do polymorphism.

**Comment by mechtaev** [comment]
Not sure of dynamic typing is suitable for Datalog. In Prolog, exceptions are thrown when an argument is of a wrong type. If we just implement logical failures, it might be hard to debug.
